aws_jupyter_proxy/awsconfig.py

```diff
@@ -0,0 +1,19 @@
+from notebook.base.handlers import IPythonHandler, utcnow
+import json
+from botocore.session import get_session
+
+class AwsConfigHandler(IPythonHandler):
+    def get(self):
```

Comment:
Could we make this `async` as Tornado is a single-threaded web server? https://github.com/aws/aws-jupyter-proxy/blob/d67537d641f94c91eb92ee0aa507b65a327a60b1/aws_jupyter_proxy/awsproxy.py#L97